### PR TITLE
test: does not have 100% coverage

### PR DESCRIPTION
**Pull request by edwaraco**
- feat: Add new dispatch in test for get default bloomrun.
- fix: Create new validation for get default bloomrun.

### CODE REVIEW

File: test/test.unit.js [hunk 1]
```diff
@@ -45,4 +45,5 @@ test('unit: test .handle() and .dispatch()', function(t) {
 	redbloom.dispatch({ns: 'test', cmd: 'w/ payload', payload: 'payload'});
 	redbloom.dispatch({ns: 'test', cmd: 'w/o payload'});
 	redbloom.dispatch({ns: 'test'});
+	redbloom.dispatch({ns: 'cmd'});
```
Comment (bbartolome) [inline test/test.unit.js:48]: @earayo, you're on the right track that there should be a dispatch here for an action that was not registered.

However, it doesn't seem to test for the expected behaviour that if an unregistered action was dispatch, it should not change the current state.

If you can update this pull request to reflect the expected behaviour, I'll be happy to merge it. Thanks!

File: index.js [hunk 2]
```diff
@@ -26,9 +26,16 @@ function redbloom(initialState, options) {
...
+				if (listActions.length == 0) {
+					listActions.push(bloomrun.lookup(action))
+				}
+				return listActions;
+			}
 			return Rx.Observable
-				.from(bloomrun.list(action))
-				.map(reducer => reducer.bind(observable, action));
+				.from(listActions())
+				.map(reducer =>	reducer.bind(observable, action));
```
Comment (bbartolome) [inline index.js:36]: Additionally, the changes here are not necessary as it was behaving as expected which was to provide a default reducer function if none was found.

From what I could tell, it looks like these changes the behaviour by pushing the default reducer twice.

**Review by bbartolome — CHANGES_REQUESTED** [review]
Thanks for the PR! I'll update the related issue to include the expected behaviour if somebody else would like to take over.

File: test/test.unit.js (hunk 1 above)
Comment (edwaraco) [reply test/test.unit.js:48]: @bbartolome Can you give me an example please?. Thanks!

**Comment by edwaraco** [comment]
@bbartolome you can check again please. Thanks!

**Review by bbartolome — APPROVED** [review]

**Comment by bbartolome** [comment]
@earayo Thanks for the submission! It looks good and I've merged it. I apologize if it took a while. I've been recently focusing a lot on my other projects and I've just managed to catch up.

Thanks again!

**Pull request merged**